types/cheerio/index.d.ts

```diff
@@ -263,6 +263,7 @@ interface CheerioElement {
     parent: CheerioElement;
     parentNode: CheerioElement;
     nodeValue: string;
+    data: string;
```

Comment:
Should this be optional? I'm concerned that this is a breaking change for consumers of this API.

Reply:
I have tested. It seem to be a option attribute.